docs/2.9.0/docs/canton/usermanual/support_checklist.rst

```diff
@@ -0,0 +1,138 @@
+..
+   Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates.
+..
+   Proprietary code. All rights reserved.
+
+.. _support_checklist:
+
+Support Checklist
+=================
+
+So that we can successfully provide support, and to ensure that you can troubleshoot the system
+efficiently on your own, we recommend that you follow the checklist below to ensure that you have access to
+all the necessary information and tools to resolve an issue and that you have established the necessary administrative
+access to your production system.
+
+The :ref:`troubleshooting guide <troubleshooting_guide>` goes into detail on how to use the information and tools
+provided in this checklist.
+
+Please note that the below list is used to prepare for the worst-case scenario and generally refers to good
+practices for being prepared for any kind of issue.
+
+.. _support_remote_console:
+
+Remote Console Access
+---------------------
+
+* **Admin Console Access**
+
+    You are able to connect to each Canton node in your setup using a :ref:`remote console <canton_remote_console>`.
+
+* **Check Node Health**
+
+    Using the admin console, you can run `node.health.status` to check the health of the node.
+
+* **Download the Health Dump**
+
+    Using the admin console, you can run `node.health.dump()` to download a :ref:`health dump <creating_dumps>` of the node.
+    Note that for versions prior to 2.9, you might need to add a ``timeout=10.minutes`` argument to the command.
+
+        * Confirm that you can download the health dump.
+        * Confirm that you can upload the health dump to a support ticket.
+
+* **Smoke Test**
+
+    Using the admin console on a participant, you can ping the participant to run an end-to-end check:
+    ``participant.health.ping(participant, id="READY-FOR-PRODUCTION")``
+
+* **Ledger API Access**
+
+    Using the console on a participant, you can access the Ledger API for further troubleshooting.
+    For this, you need to be able to enable the flag ``canton.features.enable-testing-commands = yes``
```

Comment:
Do we want to remove the need of testing commands to access LAPI? It feels wrong to me that in standard instructions (here, getting started guide) you need testing commands enabled.

Reply:
Yes, in 2.x we should remove this. This is historic as there was the request to not suggest to use the console for interaction with the ledger api ...